package.json

```diff
@@ -20,6 +20,7 @@
     "ci-test": "npm run format && npm test",
     "start-sparkswapd": "pm2-docker pm2.json --watch",
     "broker-proto": "pbjs -t proto3 broker.proto -p ./broker-daemon/proto -o /dev/null && cp ./broker-daemon/proto/broker.proto ./broker-cli/proto/broker.proto",
+    "validate-config": "standard ~/.sparkswap.js",
```

Comment:
i don't think this actually validates the config...

Reply:
@treygriffith what is your hesitation? The screenshot in the description is an example of it working against a bad file

Reply:
@treygriffith I think I understand what you mean. This is one type of validation, however ive added a card that can help us actually write a 'validator' for this (if rpcAddress exists, if whatever keys exist)